prisma/migrations/20230111224413_project_board/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- DropIndex
+DROP INDEX "profiles_search_col_idx";
+
+-- DropIndex
+DROP INDEX "projects_ts_column_idx";
+
+-- AlterTable
+ALTER TABLE "Projects" ADD COLUMN     "projectBoard" TEXT,
+ALTER COLUMN "tsColumn" DROP DEFAULT;
```

Comment:
I think this file can be left with only these lines. The other ones that modify the indexes should not be added, since you'll be removing the full text indexes we are using for the project searches.

This may fix the deploy and testing issues.

Reply:
Didn't work 😢